org.bino3d.bino.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.bino3d.bino
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+command: bino
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Play videos from the net
+  - --share=network
+  # OpenGL rendering
+  - --device=dri 
+  # Webcam capturing
+  - --device=all
+  # Audio output
+  - --socket=pulseaudio
+  # Play video files, but only read access required
+  - --filesystem=host:ro
+cleanup:
+  - '/include'
+  - '/lib/*.a'
+  - '/lib/cmake'
+  - '/lib/debug'
+modules:
+  - name: openvr
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://github.com/ValveSoftware/openvr/archive/refs/tags/v1.14.15.tar.gz
+        sha256: 40644e569f057e86c733d818a22d821c640307e5058d95958414422134d79584
+    cleanup:
+      - '/share/pkgconfig'
+  - name: vrpn
+    buildsystem: simple
+    build-commands:
+      - cmake -B build --install-prefix=/app -DCMAKE_BUILD_TYPE=Release
+      - cmake --build build
+      - cmake --install build
+    sources:
+      - type: archive
+        url: https://github.com/vrpn/vrpn/archive/refs/tags/version_07.35.tar.gz
+        sha256: faa98bd6f81be2ddaaf438fc3a89cfcc929a67627ddb72a3d4fe795fcf51de29
+    cleanup:
+      - '/bin'
+      - '/etc'
+      - '/lib/python*'
+      - '/share/vrpn-07.35'
+  - name: qvr
+    buildsystem: simple
+    build-commands:
+      - cmake -S libqvr -B build --install-prefix=/app -DCMAKE_BUILD_TYPE=Release
```

Comment:
why not using `cmake-ninja` buildsystem?

Reply:
With QVR the problem is that the CMakeLists.txt file is in the libqvr subdirectory, not the root directory, and simply passing the `-S libqvr` option in `config-opts` does not work for some reason; I had to specify the full command.
But I was able to change the generator to ninja.

Reply:
`subdir` is here for that.

Reply:
Again, thanks for that. Fixed now and works with `cmake-ninja`.